operator/gitops/argocd/pipeline-service/tekton-results/base/kustomization.yaml

```diff
@@ -26,3 +26,28 @@ patchesJson6902:
         path: "/metadata/labels"
         value:
           argocd.argoproj.io/managed-by: openshift-gitops
+  - target:
+      kind: Service
+      name: tekton-results-api-service
+      namespace: tekton-results
+    patch: |-
+      - op: add
+        path: "/metadata/annotations"
+        value:
+          service.beta.openshift.io/serving-cert-secret-name: tekton-results-tls
+  - target:
+      kind: Deployment
+      name: tekton-results-api
+      namespace: tekton-results
+    patch: |-
+      - op: replace
+        path: /spec/template/spec/containers/0/env/2/value
+        value: tekton-results-postgres-service.tekton-results.svc.cluster.local
```

Comment:
We need to avoid hardcoding the host of the DB here.
It is going to work in dev env but currently it is not going to work in the staging/production env where the right Host is provided in the tekton-results-databse secret

Reply:
done